src/frontend/screens/AddToProjectScreen/index.tsx

```diff
@@ -9,27 +10,49 @@ import { MEDIUM_BLUE, WHITE } from "../../lib/styles";
 import HeaderTitle from "../../sharedComponents/HeaderTitle";
 import { BackIcon } from "../../sharedComponents/icons";
 import IconButton from "../../sharedComponents/IconButton";
+// TODO: Make this a shared component instead?
 import { WithWifiBar } from "../Onboarding/WithWifiBar";
 
-import { ScanQrCodeStep } from "./ScanQrCodeStep";
+import { ConnectStep } from "./ConnectStep";
 import { DeviceFoundStep } from "./DeviceFoundStep";
+import { ScanQrCodeStep } from "./ScanQrCodeStep";
 import { SuccessStep } from "./SuccessStep";
 
-type Step = "scan" | "found" | "success";
+type Step = "scan" | "found" | "connect" | "success";
 
 const m = defineMessages({
   titleGeneric: {
-    id: "screens.Onboarding.AddToProjectScreen.titleGeneric",
+    id: "screens.AddToProjectScreen.titleGeneric",
     defaultMessage: "Add to Project",
   },
 });
 
+// TODO: This will make some API call and wait for a response from the invited device
+// Resolve if accepted, reject if declined?
+const sendInviteToDevice = (deviceId: string) =>
+  new Promise((res, rej) => {
+    setTimeout(res, 2000);
+  });
+
 export const AddToProjectScreen: NavigationStackScreenComponent = () => {
+  const navigation = useNavigation();
+
   const [screenLoaded, setScreenLoaded] = React.useState(false);
   const [step, setStep] = React.useState<Step>("scan");
   const [foundDeviceId, setFoundDeviceId] = React.useState<string>();
   const [config] = React.useContext(ConfigContext);
 
+  const sendInvite = async (deviceId: string) => {
+    try {
+      setStep("connect");
+      await sendInviteToDevice(deviceId);
+      setStep("success");
+    } catch (err) {
+      // TODO: provide some messaging about failure state
+      setStep("found");
+    }
```

Comment:
We need some designs for this. I'm assuming at the very least we can bring the user back to the found step, but maybe we'd want a separate screen/step if the invited device declined the invite. Will ping Sabella